src/UI/Components/home_header/index.tsx

```diff
@@ -0,0 +1,28 @@
+
+import { View, Text, Image, StyleSheet, ImageSourcePropType } from 'react-native';
+import AvatarCircle from '../AvatarCircle';
+import { styles } from './styles';
+
+interface HeaderProps {
```

Comment:
why are we using `interface`? Please read once why `interface` is used. use `type` here instead, and add docs here